### PR TITLE
Add Census Race Definitions to Methodology Page

### LINKED ISSUE #1720: Add Census Race group definitions to methodology page
**Describe the bug**
Currently, the `Native Hawaiian/ Pacific Islander (Non-Hispanic)` and `American Indian/ Alaska Native (Non-Hispanic)` races are not included on our methodology page's examples of hispanic/non-hispanic races. We always strive to include small demographic groups on the tracker and those two racial groups should be included when we update to include a definitions of races

The section of the methodology in question was not designed as a glossary to explain the races, but as a list of examples to help users understand Hispanic inclusive versus Non-Hispanic races. However, it's very confusing and since we DON'T have a glossary yet, this has become the de facto race terminology glossary. It's lacking MANY race/eth combos, and we should address the above original concerns as we redesign our #1706 

**To Reproduce**
1. Go to https://healthequitytracker.org/methodology
2. Scroll down to the bottom and look at the `What do the race/ethnicity groups mean?` sections
3. See that AIAN and NHPI are not included

**Expected behavior**
As a user of the site curious about what the race/ethnicity groups mean I can 
1. Go to https://healthequitytracker.org/methodology
2. Scroll down to the bottom and look at the `What do the race/ethnicity groups mean?` sections
3. I can see a definition for `Native Hawaiian/ Pacific Islander (Non-Hispanic)`
4. I can see a definition for `American Indian/ Alaska Native (Non-Hispanic)`

**Screenshots**

Screen Shot 2022-07-27 at 2 34 05 PM

**Additional context**

Make sure to get an approval of the definitions from JC or Nelson

### PR DESCRIPTION
**Pull request by eriwarr**
## Preview Link

<a href="https://deploy-preview-1736--health-equity-tracker.netlify.app/methodology">
  <br />
Click to demo updates </a> 

## Description

This PR updates the existing race descriptions for `Asian`, `Black or African American`,  and `Two or more races` to the census definition; and adds the census definitions for `American Indian and Alaska Native`, `Native Hawaiian or Other Pacific Islander`, and `White.` 

## Motivation and Context

closes #1720

## How has this been tested?

<a href="https://www.census.gov/quickfacts/fact/note/US/RHI625221">Long Descriptions</a>
<a href="https://www.census.gov/topics/population/race/about.html">Short Descriptions</a>

## Screenshots (if appropriate):
Screen Shot 2022-08-08 at 2 36 57 PM

## Types of changes

- New feature (non-breaking change which adds functionality or content)

### CODE REVIEW

File: frontend/src/pages/DataCatalog/MethodologyTab.tsx [hunk 1]
```diff
@@ -716,16 +716,44 @@ function MethodologyTab() {
...
+                  </li>
+                  <li>
+                    <b>Two or more races (Non-Hispanic)</b>: Combinations of two
+                    or more of the following race categories: "White," "Black or
+                    African American," American Indian or Alaska Native,"
+                    "Asian," Native Hawaiian or Other Pacific Islander," or
+                    "Some Other Race"
+                  </li>
+                  <li>
+                    <b>Two or more races & Unrepresented race (Non-Hispanic)</b>
```
Comment (benhammondmusic) [inline frontend/src/pages/DataCatalog/MethodologyTab.tsx:753]: Let's define "unrepresented race" first, using the census definition for "some other race", and then have this combo race that explains it's a combo of these two previously defined races

File: frontend/src/pages/DataCatalog/MethodologyTab.tsx [hunk 2]
```diff
@@ -716,16 +716,44 @@ function MethodologyTab() {
                     race/ethnicity.
                   </li>
                   <li>
-                    <b>Asian (Non-Hispanic)</b>: A single race (Asian), not
-                    Hispanic/Latino.
+                    <b>American Indian and Alaska Native (Non-Hispanic)</b>: A
```
Comment (benhammondmusic) [inline frontend/src/pages/DataCatalog/MethodologyTab.tsx:719]: For each of these we should either remove "(Non-Hispanic)" and then have a single entry at the bottom that explains NH races, or simply include in each entry that this NH race group excludes anyone who identifies Hispanic as their ethnicity

**Review by benhammondmusic — COMMENTED** [review]
If we make those couple tweaks I think we should merge this now, and then the updated races will be in place when we go to redesign the methodology

**Review by benhammondmusic — APPROVED** [review]
looks great. Just confirm with Josh and JC before you merge this and make sure they agree with this approach

**Review by joshzarrabi — APPROVED** [review]
Looks awesome, thank you so much Eric

**Pull request merged**